modules/OCPP201/OCPP201.cpp

```diff
@@ -473,24 +518,24 @@ bool OCPP201::all_evse_ready() {
 }
 
 void OCPP201::set_connector_operational_status(const ocpp::v201::OperationalStatusEnum operational_status,
-                                               const int32_t evse_id, const int32_t connector_id) {
+                                               const int32_t evse_id, const int32_t connector_id, const bool persist) {
     if (operational_status == ocpp::v201::OperationalStatusEnum::Operative) {
         this->evses.at(evse_id).connectors.at(connector_id) = ocpp::v201::OperationalStatusEnum::Operative;
         auto connector_id_str = std::to_string(evse_id) + "." + std::to_string(connector_id);
         this->r_kvs->call_delete(KVS_OCPP201_INOPERATIVE_KEY_PREFIX + connector_id_str);
-    } else {
+    } else if (persist) {
```

Comment:
Is it true, that the status will only be set to inoperative, when the persist flag is true?